libraries/aws/common/src/aws_iot_parser.c

```diff
@@ -145,8 +145,17 @@ AwsIotStatus_t AwsIot_ParseStatus( const char * pTopicName,
     IOT_FUNCTION_ENTRY( AwsIotStatus_t, AWS_IOT_UNKNOWN );
     const char * pSuffixStart = NULL;
 
+    /* Both 'accepted' and  'rejected' topics are of the same length
+     * The below is a defensive check at compile time to ensure that.
+     * static_assert is only available from C11 and above*/
+    #if ( __STDC_VERSION__ >= 201112L )
```

Comment:
We're targeting C99 with these libraries.  I don't think this block is useful.

Reply:
yea. makes sense. Thanks